README.md

```diff
@@ -12,6 +12,7 @@
 ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/reframe-hpc)<br/>
 [![Slack](https://reframe-slack.herokuapp.com/badge.svg)](https://reframe-slack.herokuapp.com/)<br/>
 [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+[![DOI](https://zenodo.org/badge/89384186.svg)](https://zenodo.org/badge/latestdoi/89384186)
```

Comment:
Since you have added a line on the README, please also change:
https://github.com/eth-cscs/reframe/blob/cbc5bd1b1ea010fc14cb082797917e8529b0d373/setup.py#L11

to take into account he line addition.